README.md

```diff
@@ -141,6 +141,14 @@ The default tag does not contains the latest API version. Please make sure the l
 
 To fix this error. You should add the latest API version swaggers to the default tag or change default tag to reference latest api version tag.
 
+### MULTIPLE_DEFAULT_TAGS
+
+Level: ERROR

```

Comment:
I wonder if this should be warning so far?  For Data Plane, seems multiple TAGs are supported. Or could we put on ERROR for management plane PR, WARNING for data plane PR?

Reply:
Yes, we can report ERROR as default, and if path contains `data-plane` and set to WARNING.